components/main.jsx

```diff
@@ -0,0 +1,32 @@
+import Image from "next/image";
```

Comment:
Please make all the component filenames in PascalCase. i.e make it `Main.jsx` instead of `main.jsx`